daffodil-core/src/main/scala/org/apache/daffodil/dsom/SequenceGroup.scala

```diff
@@ -260,12 +261,7 @@ abstract class SequenceGroupTermBase(
 trait SequenceDefMixin
   extends AnnotatedSchemaComponent
   with GroupDefLike
-  with ResolvesProperties //
-  // Technically, this is not a term, so shouldn't resolve scoped properties
-  // and doesn't take into account combining ref with def properties.
-  // But we need to get one local property: dfdl:hiddenGroupRef.
-  // So we mixin ResolvesProperties, but shouldn't use it except to obtain dfdl:hiddenGroupRef.
-  {
+  with FindPropertyMixin {
```

Comment:
Should this be ResolvesLocalProperties for dfdl:hiddenGroupRef instead of FindPropertyMixin?

Reply:
This gets mixed into classes which define ResolvesScopedProperties, or ResolvesLocalProperties so this use of FindPropertyMixin is just to get it to compile without errors. 

Turns out if something has ResolvesScopedProperties mixed in, that doesn't mean all its properties are scoped. It still checks the list of local-only properties like outputValueCalc for elements, and in this case for sequences the dfdl:hiddenGroupRef property, and doesn't "scope" those.  So this does the right thing whether mixed into a class that also mixes in the "scoped" or "local" flavor of property resolving.